fleet_output/main.tf

```
@@ -1,19 +1,12 @@
-resource "restapi_object" "elasticsearch_output" {
-  path         = "/api/fleet/outputs"
-  id_attribute = "item/id"
-
-  data = jsonencode({
-    name                  = var.name
-    type                  = "elasticsearch"
-    is_default            = false
-    is_default_monitoring = false
-    hosts = [
-      var.elasticsearch_url
-    ]
-    config_yaml = yamlencode({
-      ssl = {
-        verification_mode = var.insecure ? "none" : "certificate"
-      }
-    })
+resource "elasticstack_fleet_output" "elasticsearch_output" {
+  name = var.name
+  type = "elasticsearch"
+  config_yaml = yamlencode({
+    "ssl.verification_mode" : var.insecure ? "none" : "certificate"
```

I am not a big fan of the dotted YAML format that so prevalent on our configs.

```suggestion
    ssl = {
      verification_mode = var.insecure ? "none" : "certificate"
    }
```